STM32/Temperature/Core/Src/main.c

```diff
@@ -355,21 +283,19 @@ int main(void)
     /* USER CODE BEGIN 3 */
 
 	// Upload data every "tsUpload" ms.
-	if (HAL_GetTick() - timeStamp > tsUpload)
-	{
+	if (HAL_GetTick() - timeStamp > tsUpload){
```

Comment:
I am not a big fan of having '{' on the same line as the if statement. The scope of the 'if' will be a bit more clear if moved to the next (code is more readable). But, we just have to agree on a style.

Reply:
I personally prefer the other, but since you like the other better I am happy to switch :)